zeppelin-interpreter/src/test/java/org/apache/zeppelin/dep/DependencyResolverTest.java

```diff
@@ -89,13 +89,13 @@ public void testLoad() throws Exception {
     // load from added repository
     resolver.addRepo("sonatype",
         "https://oss.sonatype.org/content/repositories/agimatec-releases/", false);
-    resolver.load("com.agimatec:agimatec-validation:0.9.3", testCopyPath);
```

Comment:
Need to be modified here?

Reply:
TestLoad() execution failed causing CI to fail.

Reply:
@liuxunorg this is an accidental change. However, agimatec-validation 0.9.3 or 0.12.0 are not available and it is addressed in #3381